src/mondefense.cpp

```diff
@@ -47,6 +47,13 @@ void mdefense::zapback( monster &m, Creature *const source,
 
     const player *const foe = dynamic_cast<player *>( source );
 
+    // Players/NPCs can avoid the shock if they wear non-conductive gear on their hands
+    for( const item &i : foe->worn ) {
```

Comment:
What happens if `foe` is `nullptr` (a monster is attacking the shocker, not a player)? I think this should be under the `foe != nullptr` check.

Reply:
Forgot about that. Thanks.